plugins/postcss-design-tokens/README.md

```diff
@@ -74,13 +74,69 @@ Use `style-dictionary3` in `@design-tokens` rules to pick this format.
 
 ## Options
 
+### importAtRuleName
+
+The `importAtRuleName` option allows you to set a custom alias for `@design-tokens`.
+
+```js
+postcssDesignTokens({ importAtRuleName: 'tokens' })
+```
+
+```pcss
+@tokens url('./tokens.json') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+	padding-top: design-token('size.spacing.small');
+	padding-left: design-token('size.spacing.small' to px);
+	padding-bottom: design-token('size.spacing.small' to rem);
+}
+
+/* becomes */
+
+.foo {
+	color: #fff;
+	padding-top: 16px;
+	padding-left: 16px;
+	padding-bottom: 1rem;
+}
+```
+
 ### is
 
 The `is` option determines which design tokens are used.
 This allows you to generate multiple themed stylesheets.
 
 By default only `@design-tokens` without any `when('foo')` conditions are used.
 
+### valueFunctionName
```

Comment:
`### is` should remain right before `#### Example usage`

Reply:
Reason I moved is to keep them alphabetically sorted but I guess `is` is more important?

Reply:
`### is` is the parent chapter of `#### Example usage`.
So it's `Example usage of "is"` :)

Reply:
🤦

Reply:
Thanks for catching that!